salt/runners/virt.py

```diff
@@ -273,6 +277,7 @@ def init(
                     install,
                     pub_key,
                     priv_key,
+                    seed_cmd,
```

Comment:
I think `seed_cmd` should be called after `enable_vnc` here, yes?

I know this function call wasn't your code, but it might be wise to start passing all of these options though with kwarg assignments (like `hypervisor=hypervisor, start=start, etc) so we don't get mixed up here. Thoughts?

Reply:
No, the `seed_cmd` argument comes before `enable_vnc` in the virt module. That's the issue I'm trying to fix.

But I think you're right about kwargs. That is a way better approach so we don't get the arguments mixed up in the future. I'll take a look at cleaning it up.